values.yaml

```
@@ -79,140 +89,146 @@ global:
     # encryption key.
     secretKey: ""
 
-  # Enables TLS encryption across the cluster to verify authenticity of the
-  # servers and clients that connect. Note: It is HIGHLY recommended that you also
-  # enable Gossip encryption.
-  # See https://learn.hashicorp.com/consul/security-networking/agent-encryption
-  #
-  # Note: this relies on functionality introduced with Consul 1.4.1. Make sure
-  # your global.image value is at least version 1.4.1.
+  # Enables TLS (https://learn.hashicorp.com/tutorials/consul/gossip-encryption-secure)
```

```suggestion
  # Enables TLS (https://learn.hashicorp.com/tutorials/consul/tls-encryption-secure)
```
Since this talks about enabling TLS rather than gossip encryption specifically